### ISSUE TITLE
Don't load events information until the world has been displayed

### ISSUE THREAD

**Issue by eordano** [decentraland/unity-renderer#3800]
One of the first things the explorer is doing, are these "cover art" resources:

![image](https://user-images.githubusercontent.com/42750/208653226-98bff93f-b50a-4deb-a3d2-652847848489.png)

They are necessary as cover art for the "Events" page experience:
![image](https://user-images.githubusercontent.com/42750/208653171-9ba5262d-fcb4-4e31-a033-2a74e2185ab2.png)

We should be lazy about loading this, instead of eager. Postpone the loading of these assets until the world has been fully loaded.

**TL;DR: "Prewarming" of Discover tabs should wait for scene loading to be over.**
